.claude/skills/graphite-pr/scripts/gt-merge-cascade.sh

```
@@ -0,0 +1,101 @@
+    [ -n "$pr" ] && PR_LIST+=("$pr")
+done
+[ "${#PR_LIST[@]}" -gt 0 ] || fail "no-open-prs" "stack branches have no open PRs"
+
+# 3a. Gate on unresolved review threads. `gt merge` ignores them; we don't.
+#     One GraphQL call per PR; returns only an integer count, so context cost is tiny.
+REPO_NWO=$(gh repo view --json nameWithOwner -q .nameWithOwner 2>/dev/null) || fail "no-repo" "could not resolve owner/repo"
+REPO_OWNER="${REPO_NWO%/*}"
+REPO_NAME="${REPO_NWO#*/}"
+UNRESOLVED_DETAIL=""
+for pr in "${PR_LIST[@]}"; do
+    n=$(gh api graphql -f query='query($o:String!,$r:String!,$n:Int!){repository(owner:$o,name:$r){pullRequest(number:$n){reviewThreads(first:50){nodes{isResolved isOutdated}}}}}' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)